portal/forms/models.py

```diff
@@ -2,5 +2,28 @@
 
 
 class Form(models.Model):
+    BACKGROUND_CHECK = "background_check"
+    ENROLLMENT_FORM = "enrollment_form"
+    VOLUNTEER_FORM = "volunteer_form"
+    OTHER = "other"
 
+    FORM_TYPE_CHOICES = (
+        (BACKGROUND_CHECK, 'Background Check'),
+        (ENROLLMENT_FORM, 'Enrollment Form'),
+        (VOLUNTEER_FORM, 'Volunteer Form'),
+        (OTHER, 'Other')
+
+    )
```

Comment:
We might consider making form types another table, but let's see how the NP reacts to this on staging.